blazor/common/how-to/port-blazor-server-maui.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: Port Blazor Server Side App to MAUI Blazor Hybrid App | Syncfusion
+description: Checkout and learn here all about porting Syncfusion Blazor Server App to .NET MAUI Blazor Hybrid App.
+platform: Blazor
+component: Common
+documentation: ug
+---
+
+# How to port Syncfusion Blazor Server App to MAUI Blazor Hybrid App
+
+This section explains how to port Syncfusion Blazor Server App to .NET MAUI Blazor Hybrid App using Razor Class Library (RCL). This way, you can avoid rewriting all your Blazor Server App pages for the .NET MAUI Blazor Hybrid App.
+
+## Prerequisites
+
+[.NET 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) or later.
+
+## Create a new project for Blazor Server Side App
```

Comment:
Create a new project for Blazor Server App

Reply:
Updated as Blazor Server App